### DISCUSSION TITLE
Make error

### DISCUSSION THREAD

**Discussion by Sirashs** [AlexeyAB/darknet#7528] [General]
nvcc  -gencode arch=compute_30,code=sm_30 -gencode arch=compute_35,code=sm_35 -gencode arch=compute_50,code=[sm_50,compute_50] -gencode arch=compute_52,code=[sm_52,compute_52] -Iinclude/ -Isrc/ -DOPENCV `pkg-config --cflags opencv`  -DGPU -I/usr/local/cuda/include/ -DCUDNN  --compiler-options "-Wall -Wno-unused-result -Wno-unknown-pragmas -Wfatal-errors -fPIC -Ofast -DOPENCV -DGPU -DCUDNN" -c ./src/convolutional_kernels.cu -o obj/convolutional_kernels.o
Segmentation fault (core dumped)
Makefile:92: recipe for target 'obj/convolutional_kernels.o' failed
make: *** [obj/convolutional_kernels.o] Error 139
installed cuda 11.2
how to solve this?

**Comment by stephanecharette** [comment]
My guess:  you're out of memory or disk space.

**Comment by stephanecharette** [comment]
Buy a bigger hard drive?  Delete unnecessary files?  Grow your partition?  This is a basic computer administration thing, it has nothing to do with Darknet.
